load_fund_echo/routes.py

```diff
@@ -0,0 +1,103 @@
+import json
```

Comment:
Business logic should not be written in routes.py
routes.py only parses and validates http request and cares about response in terms of success or failure and their respective http codes.